src/worker/abstract-worker.ts

```diff
@@ -3,6 +3,10 @@ import type { Worker } from 'cluster'
 import type { MessagePort } from 'worker_threads'
 import type { MessageValue } from '../utility-types'
 import type { WorkerOptions } from './worker-options'
+import { killBehaviorEnumeration } from './worker-options'
+
+const defaultMaxInactiveTime = 1000 * 60
+const defaultKillBehavior = killBehaviorEnumeration.SOFT
```

Comment:
```suggestion
const defaultKillBehavior: 'HARD' | 'SOFT' = killBehaviorEnumeration.SOFT
```

Reply:
I don't think is really needed

Reply:
Could you please help with the above error? Also applying your suggestions does not work.

[!] (plugin rpt2) Error: /Users/alessandropioardizio/git/node-pool/src/worker/abstract-worker.ts(9,7): semantic error TS2322: Type 'string' is not assignable to type '"HARD" | "SOFT"'.
src/worker/abstract-worker.ts
Error: /Users/alessandropioardizio/git/node-pool/src/worker/abstract-worker.ts(9,7): semantic error TS2322: Type 'string' is not assignable to type '"HARD" | "SOFT"'.

Reply:
Moreover with your suggestions we are duplicating the 'HARD' and 'SOFT' string in much places how we can handle that better in your opinion ?